packages/apps/job-launcher/server/src/common/enums/webhook.ts

```diff
@@ -1,7 +1,10 @@
 export enum EventType {
   ESCROW_CREATED = 'escrow_created',
   ESCROW_CANCELED = 'escrow_canceled',
+  TASK_REJECTED = 'task_rejected',
   TASK_CREATION_FAILED = 'task_creation_failed',
+  TASK_COMPLETED = 'task_completed',
+  TASK_FINISHED = 'task_finished'
```

Comment:
@eugenvoronov why did you add these events? Not sure we need to know about event types that are actually shared between exchange and recording oracles only.

Reply:
@Dzeranov  I have decided to bring this enum to one type in the services where it is used. Basically, if you are sure that none of these events will be sent to the launcher, we can delete them.